src/pages/_app.tsx

```diff
@@ -0,0 +1,48 @@
+import { initializeIcons } from '@fluentui/react'
```

Comment:
How does this work? I thought next.js required this file to be in `/pages/_app.tsx`?

Reply:
do you mean it being under `src`? The docs mentioned that next is configured to use `src` to wrap everything or just have `pages` at the top level (since `src` is a common pattern across the JS community).... since we use the `src` pattern, I thought it was nice to align to that